edx_event_bus_kafka/tests/test_config.py

```diff
@@ -0,0 +1,50 @@
+"""
+Test common configuration loading.
+"""
+
+from unittest import TestCase
+
+from django.test.utils import override_settings
+
+from edx_event_bus_kafka import config
+
+try:
+    from confluent_kafka.schema_registry import SchemaRegistryClient
+except ImportError:  # pragma: no cover
+    SchemaRegistryClient = None  # pragma: no cover
+
+
+class TestSchemaRegistryClient(TestCase):
+    def test_unconfigured(self):
+        assert config.create_schema_registry_client() is None
+
+    def test_configured(self):
+        with override_settings(EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL='http://localhost:12345'):
+            assert isinstance(config.create_schema_registry_client(), SchemaRegistryClient)
```

Comment:
Does this test only work if the library is installed? If not, should we just pass in None or a mock, without the import that could fail?

Reply:
It only works if the library is installed, but `test.in` does include it.